### PR TITLE
Monaco Component Support Model Value Binding

### PR DESCRIPTION
**Pull request by Kinplemelon**
- extract the rule outputs component
- updated the props of monaco & fixed a bug

### CODE REVIEW

File: src/components/Monaco.vue [hunk 1]
```diff
@@ -3,100 +3,108 @@
...
+    default: false,
+  },
+  // warp: {
+  //   type: Boolean,
+  //   default: false,
+  // },
+  // provider: {
+  //   type: Array,
+  //   default: () => [],
+  // },
```
Comment (ysfscream) [inline src/components/Monaco.vue:41]: Should we need these options? I think we should.

File: src/components/Monaco.vue [hunk 2]
```diff
@@ -3,100 +3,108 @@
...
-      }
+const emit = defineEmits(['update:modelValue'])
 
-      editor.value = monaco.editor.create(document.getElementById(id), defaultOptions)
+// ❗️ editor instance can not be reactive, otherwise it will cause the page to get stuck for unknown reasons
+let editor = {}
+// const editValue = computed(() => {
+//   console.log(prop.modelValue + "changed");
+//   return prop.modelValue;
+// });
```
Comment (ysfscream) [inline src/components/Monaco.vue:59]: OK, It shouldn't matter if you don't use reactive here, what does the pseudo-code comment mean?

File: src/i18n/RuleEngine.js [hunk 3]
```diff
@@ -355,6 +359,14 @@ export default {
     zh: 'SQL 语句',
     en: 'SQL',
   },
+  ruleSQLDesc: {
+    zh: '使用 SQL 语句实时提取、过滤、丰富和转换设备与业务系统之间的 IoT 数据。',
+    en: 'Extract, filter, enrich, and transform IoT data between devices and business systems in real-time using SQL statements.',
+  },
+  doc: {
+    zh: '参考文档',
+    en: 'Doc',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:368]: Document?

**Review by ysfscream — APPROVED** [review]

File: src/i18n/RuleEngine.js (hunk 3 above)
Comment (Kinplemelon) [reply src/i18n/RuleEngine.js:368]: ok

File: src/components/Monaco.vue (hunk 2 above)
Comment (Kinplemelon) [reply src/components/Monaco.vue:59]: It's the previous code, I always don't want to delete the previously commented code, what if I need it later?

File: src/components/Monaco.vue (hunk 1 above)
Comment (Kinplemelon) [reply src/components/Monaco.vue:41]: I haven't figured out the usage of these two yet, I'll add them later.

File: src/components/Monaco.vue (hunk 2 above)
Comment (ysfscream) [reply src/components/Monaco.vue:59]: It looks like for debugging... Ok, just remove it?

Comment (ysfscream) [reply src/components/Monaco.vue:59]: BTW, I think the init data of the `editor` will be `null`?

**Pull request merged**